tests/test_helper.tcl

```diff
@@ -101,6 +101,7 @@ set ::all_tests {
     unit/cluster/multi-slot-operations
     unit/cluster/slot-ownership
     unit/cluster/links
+    unit/client
```

Comment:
i see we already have quite a few tests for the CLIENT sub-commands in tests/unit/introspection.tcl, so now sure about adding another unit, specifically when all it has are error cases.

Reply:
yes its true, deleted the file and added the tests next to client tests in introspect.tcl